build/Glyssen.proj

```diff
@@ -59,11 +59,11 @@
 	<CreateProperty Value="$(Version).$(BuildHash)">
 		<Output PropertyName="FullVersion" TaskParameter="Value"/>
 	</CreateProperty>
-	<CreateProperty Value="GlyssenInstaller.$(Version).exe" Condition=" '$(Label)' == '' ">
-		<Output PropertyName="InstallerFileName" TaskParameter="Value"/>
+	<CreateProperty Value="$(RootDir)\output\installer\GlyssenInstaller.$(Version).exe" Condition=" '$(Label)' == '' ">
+		<Output PropertyName="InstallerPathWithVersion" TaskParameter="Value"/>
```

Comment:
This is ok. However, since touching these lines it might be better to refactor them: msbuild now allows to use `PropertyGroup` inside of targets, so there's no longer a need to use `CreateProperty`. IMO this would make it easier to read.

```
<PropertyGroup>
    <FullVersion>$(Version).$(BuildHash)</FullVersion>
    etc.
</PropertyGroup>
```

Reply:
Done